01-introduction.Rmd

```diff
@@ -55,9 +54,6 @@ non-spatial attribute data are inevitably interwoven with geometry data, and Geo
 The latter communicates that this is a book about using R as a GIS, to perform spatial operations on *geographic data* [@bivand_applied_2013].
 However, the term GIS conveys some connotations (see Table \@ref(tab:gdsl)) which simply fail to communicate one of R's greatest strengths:
 its unparalleled console-based ability to seamlessly switch between geographic and non-geographic data processing, modeling and visualization tasks.
-<!--
-"R for GIScience" or "GIScience with R" would have not implied GUI-based geoprocessing. The same goes for "Geo-data science with R"; maybe we should point out more prominently that Leeds was the birth place of geocomputation, and therefore the natural choice for the title :)
--->
 By contrast, the term geocomputation implies reproducible and creative programming.
```

Comment:
I'm up for that :+1: Although I should say I was not in Leeds when all the Stan Opensaw-inspired stuff was kicking off. People including @agdturner and @AJEvans were though and may have something to say about this! I think the Leeds link is interesting but not crucial to the book. Plus I don't want to be parochial: I think Geocomputation is a useful term regardless of your school of thought or where you're coming from. Happy for it to go a little further in that direction though.